Tribler/Core/Utilities/bootstrap_util.py

```diff
@@ -1,5 +1,9 @@
 from __future__ import absolute_import
 
+import random
```

Comment:
Why are these imports added here? The `random` module is not used (anymore?) in this file. Maybe there's a rebase issue?

Reply:
Probably so. I haven't run the pre-commit hooks after the rebase.
Don't worry about this small discrepancies. When we'll run the hooks on the whole repository, that will cleanup everything.